### PR TITLE
Mark a book as read

### PR DESCRIPTION
**Pull request by phanbt58**

### CODE REVIEW

File: app/controllers/marks_controller.rb [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
+    end
+    redirect_to @book
+  end
+
   private
   def marks_params
     params.require(:mark).permit :user_id, :book_id, :mark_type
   end
+
+  def set_mark
```
Comment (hieuhtk58) [inline app/controllers/marks_controller.rb]: Nen de load_mark

File: app/controllers/marks_controller.rb [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+    @user_book = Mark.find_by id: params[:id]
+  end
+
+  def set_book
```
Comment (hieuhtk58) [inline app/controllers/marks_controller.rb]: Nen de load_book

**Comment by hieuhtk58** [comment]
ok

**Comment by ngotungson** [comment]
OK

**Pull request merged**